d2l-image.js

```diff
@@ -16,66 +38,71 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-image">
 				border-radius: var(--d2l-image-border-radius);
 				object-fit: var(--d2l-image-object-fit);
 			}
-		</style>
-
-		<iron-ajax id="imageRequest" url="[[imageUrl]]" headers="[[_headers]]" on-iron-ajax-response="_onImageResponse" on-iron-ajax-error="_onImageResponse" handle-as="blob">
-		</iron-ajax>
-
-		<img id="image" alt$="[[alternateText]]">
-	</template>
-
-</dom-module>`;
-
-document.head.appendChild($_documentContainer.content);
-
-Polymer({
-	is: 'd2l-image',
-
-	properties: {
-		alternateText: {
-			type: String,
-			reflectToAttribute: true,
-		},
-		imageUrl: {
-			type: String,
-			value: null
-		},
-		token: String,
-		_headers: String
-	},
-
-	observers: [
-		'_onImageUrlChange( imageUrl, token )'
-	],
-
-	_onImageUrlChange: function(imageUrl, token) {
-		if (imageUrl && token) {
-			this._headers = {
-				Authorization: 'Bearer ' + token
-			};
-			this.$.imageRequest.generateRequest();
-		} else if (imageUrl) {
-			this.$.image.src = imageUrl;
+		`;
+	}
+
+	constructor() {
+		super();
+		this.imageUrl = undefined;
+		this.token = undefined;
+		this._loadImage();
```

Comment:
This call may not be necessary, since I think `updated` is always going to get called with the values and in the constructor here `token` is `undefined` anyway so it won't do anything.